sympy/geometry/polygon.py

```diff
@@ -402,6 +403,59 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+    def second_moment_of_area(self, p=None):
+        """Second and product moment of area of polygon.
+
+        Parameters
+        ==========
+
+        p : Point, None
+            p is the point about which second moment of area is to be find. Default value is None
+
+        Notes
+        =====
+
+        function will return second moment of area about the axis passing through centroid
+        if point is not given othervise about the axis passing through the p point.
```

Comment:
otherwise

Reply:
should I use else instead of otherwise?

Reply:
otherwise is mispelled

Reply:
Also, make these complete sentences with proper punctuation and capitalization.